.github/ISSUE_TEMPLATE.md

```diff
@@ -1,6 +1,35 @@
-Please include the following information in your ticket.
+Please [read these guidelines](http://ibm.biz/cdt-issue-guide) before opening an issue.
 
-- java-cloudant-cache version(s) that are affected by this issue.
-- Java version (including vendor and platform).
-- If you're using the optional okhttp dependency.
-- A small code sample that demonstrates the issue.
+<!-- Issues will be CLOSED IMMEDIATELY if the following template is not completed. -->
+
+## Bug Description
+
+### 1. Steps to reproduce and a small code sample that demonstrates the issue
+<!--
+EXAMPLE:
+1. Initialize client
+2. List DBs
+
```

Comment:
It might be better to have steps related to cache like:
```
1. Create a new DatabaseCache with the database and cache instances.
2. Get document with ID "abcdef" from the cache

Database cachedDb = new DatabaseCache(db, cache);
MyDocument abc = cachedDb.find(MyDocument.class, "abcdef");
```

Reply:
As agreed, leaving as pseudo-code